Sources/BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -377,21 +377,8 @@ extension KeyringStore: BraveWalletKeyringServiceObserver {
 
   public func keyringCreated(_ keyringId: String) {
     Task { @MainActor in
-      var coin: BraveWallet.CoinType = .eth
-      switch keyringId {
-      case BraveWallet.DefaultKeyringId:
-        coin = .eth
-      case BraveWallet.SolanaKeyringId:
-        coin = .sol
-      case BraveWallet.FilecoinKeyringId:
-        coin = .fil
-      default:
-        break
-      }
-      
       let newKeyring = await keyringService.keyringInfo(keyringId)
       if let newAccount = newKeyring.accountInfos.first {
-        walletService.setSelectedCoin(coin)
```

Comment:
coin type will be set to the new created keyring's coin type, so we dont need to set it for the WalletService.